web-www/components/templates/Layout/Layout.tsx

```diff
@@ -49,6 +53,8 @@ const Layout = ({ children }: Props) => {
               </Body>
             </>
           }
+          requiredMessage={REQUIRED_MESSAGE}
```

Comment:
needs translation with `_(...)`?

Reply:
It's in `web-www`; as discussed, it's out of scope for now.